src/main/cljs/dmohs/react.cljs

```diff
@@ -57,11 +64,20 @@
 
 
 (defn find-dom-node
-  "See: https://facebook.github.io/react/docs/top-level-api.html#reactdom.finddomnode"
+  "See: https://reactjs.org/docs/react-dom.html#finddomnode"
   [instance]
   (core/find-dom-node instance))
 
 
+(defn create-portal
+  "See: https://reactjs.org/docs/portals.html"
+  [child container]
+  (core/create-portal (if (seqable? child)
```

Comment:
Why this check? Is there a reason to have multiple usage models for this function?

Reply:
I'd love to get rid of it, but otherwise it insists the child is a list and needs keys. I couldn't figure out a root cause.

Reply:
Do you have a branch I could check out somewhere to poke at this a bit?

Reply:
To see it, create a portal and have its contents be an element declared with brackets: `[:div {} "foo"]` rather than just feeding it `:div {} "foo"]`